src/migration/1694660281384-migration.ts

```diff
@@ -0,0 +1,79 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migration1694660281384 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const schemaName = process.env.NODE_ENV === 'test' ? 'test' : 'public';
+        // create access status
+        await queryRunner.query(
+            `CREATE TYPE ${schemaName}.access_request_status AS ENUM ('NotGranted', 'Granted', 'Rejected');`,
+        );
+
+        // create request table
+        await queryRunner.query(`CREATE TABLE IF NOT EXISTS ${schemaName}."access_request" (
+        request_id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
+        user_guid VARCHAR(36) NOT NULL,
+        identity_type VARCHAR(10) NOT NULL,
+        requested_role ${schemaName}.role_type NOT NULL,
+        organization VARCHAR(50) NOT NULL,
```

Comment:
They wouldn't neccessarily have an organization if they were BCGov, right?

Reply:
Yup they will . We have an editable text field which is a required field for identity type IDIR and read only for BCeID.